src/Database.php

```diff
@@ -2,25 +2,15 @@
 
 class Database
 {
-
-    private $databaseHost = '';
-    private $databaseUser = '';
-    private $databasePassword = '';
-    private $databaseName = '';
     private $DB;
-
     /**
      * Try to connect to database
      * @method __construct
      */
     public function __construct()
     {
         try {
-            $this->DB = new PDO(
-                'mysql:host=' . $this->databaseHost . ';dbname=' . $this->databaseName,
-                $this->databaseUser,
-                $this->databasePassword
-            );
+            $this->DB = new PDO('mysql:host='. getenv('DATABASE_HOST') .';dbname='. getenv('DATABASE_NAME'), getenv('DATABASE_USER'), getenv('DATABASE_PASSWORD'));
```

Comment:
Breaks normal setup

Reply:
I have a couple ideas for how to make this work for both normal setup and docker. 

I think using a `.env` file with the [phpdotenv](https://github.com/vlucas/phpdotenv) library would be the most ideal and secure, but would take some work. Going this route we could have a `.env.example` file the end-user renames and defines with their credentials, then include `.env` in .gitignore to prevent future accidental commits of the credentials. This solution would work for both the normal setup and with docker.

However I can implement a temporary solution without external libraries for now and we can reassess this after docker support has been merged.